README.md

```diff
@@ -190,29 +187,29 @@ config :my_app, MyApp.Mailer,
 ```
 
 You can create new adapters for any environment by implementing the
-[`Bamboo.Adapter`] behaviour.
+[`Bamboo.Adapter`] behavior.
```

Comment:
The previous spelling was the correct one. Behaviours (with the `u`) are part for [Elixir and Erlang](https://hexdocs.pm/elixir/1.3.4/behaviours.html). So we want to keep the other spelling to communicate that we mean an Elixir/Erlang behaviour.